modules/landing-page/landing-dashboard-overview.tsx

```
@@ -0,0 +1,346 @@
+      <div className="w-64 border-r border-gray-200 bg-gray-50">
+        <div className="p-4">
+          <div className="mb-6 flex items-center space-x-3">
+            <div className="flex h-8 w-8 items-center justify-center rounded-md bg-black">
+              <Zap className="h-4 w-4 text-white" />
+            </div>
+            <span className="font-medium text-black">{appConfig.appName}</span>
+          </div>
+
+          <nav className="space-y-1">
+            <a
+              href="#"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)